src/components/SideMenu/index.tsx

```diff
@@ -85,7 +85,7 @@ export function SideMenu({
     >
       {links.map((link, index) => {
         return (
-          <div key={`link_${index}`}>
+          <div key={`link_${link}`}>
```

Comment:
The key should not be the entire object. Also there could be 2 SideMenu items with the same url.